packages/system/cilium/Makefile

```diff
@@ -1,19 +1,12 @@
-NAMESPACE=cozy-cilium
 NAME=cilium
+NAMESPACE=cozy-$(NAME)
 
-show:
-	kubectl get hr -n cozy-cilium cilium -o jsonpath='{.spec.values}' | helm template --dry-run=server -n $(NAMESPACE) $(NAME) . -f -
-
-apply:
-	kubectl get hr -n cozy-cilium cilium -o jsonpath='{.spec.values}' | helm upgrade -i -n $(NAMESPACE) $(NAME) . -f -
-
-diff:
-	kubectl get hr -n cozy-cilium cilium -o jsonpath='{.spec.values}' | helm diff upgrade --allow-unreleased --normalize-manifests -n $(NAMESPACE) $(NAME) . -f -
+include ../../hack/package-system.mk
 
 update:
 	rm -rf charts
-	helm repo add cilium https://helm.cilium.io/
-	helm repo update cilium
-	helm pull cilium/cilium --untar --untardir charts --version 1.14
-	sed -i -e '/Used in iptables/d' -e '/SYS_MODULE/d' charts/cilium/values.yaml
+	helm repo add $(NAME) https://helm.cilium.io/
+	helm repo update $(NAME)
+	helm pull $(NAME)/$(NAME) --untar --untardir charts --version 1.14
+	sed -i -e '/Used in iptables/d' -e '/SYS_MODULE/d' charts/$(NAME)/values.yaml
 	patch -p3 --no-backup-if-mismatch < patches/fix-cgroups.patch
```

Comment:
I would rather keep update target without using variables, because commands here targets to concrete repositories, they are node depended on release name or namespace.